packages/store/src/decorators/select/symbols.ts

```diff
@@ -0,0 +1,31 @@
+import { Observable } from 'rxjs';
+
+import { SelectFactory } from './select-factory';
+import { META_KEY, NgxsConfig } from '../../symbols';
+import { propGetter } from '../../internal/internals';
+import { CONFIG_MESSAGES, VALIDATION_CODE } from '../../configs/messages.config';
+
+export function createSelect(fn: any): Observable<any> {
```

Comment:
this function should be in the `select.ts` file as it's used only there

Reply:
done